build.fsx

```diff
@@ -127,6 +130,60 @@ Target "MultiNodeTests" (fun _ ->
 )
 
 //--------------------------------------------------------------------------------
+// NBench targets 
+//--------------------------------------------------------------------------------
+
+// Filter out assemblies which can't run on Linux, Mono, .NET Core, etc...
+let filterPlatformSpecificAssemblies (assembly:string) =
```

Comment:
@alexvaluyskiy this should have been used for `RunTests` as well - per `dev/build.fsx`, but we are not there yet with `RunTests` due to cherry picking required by .NET Core build right now.  This is consistent with the filtering being done on `dev` branch.